api/specs/storage/scripts/storage.py

```diff
@@ -0,0 +1,377 @@
+""" Helper script to generate OAS automatically
+"""
+
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+
+import itertools
+from enum import Enum
+from typing import Any
+
+from fastapi import FastAPI, status
+from models_library.api_schemas_storage import (
+    FileMetaDataGet,
+    FileUploadCompleteFutureResponse,
+    FileUploadCompleteResponse,
+    FileUploadCompletionBody,
+    FileUploadSchema,
+    FoldersBody,
+    HealthCheck,
+    LinkType,
+    PresignedLink,
+    SoftCopyBody,
+    TableSynchronisation,
+)
+from models_library.app_diagnostics import AppStatusCheck
+from models_library.generics import Envelope
+from models_library.projects_nodes import NodeID
+from models_library.projects_nodes_io import LocationID, StorageFileID
+from models_library.users import UserID
+from pydantic import AnyUrl, ByteSize
+from servicelib.long_running_tasks._models import TaskGet, TaskId, TaskStatus
+from settings_library.s3 import S3Settings
+from simcore_service_storage._meta import api_vtag
+from simcore_service_storage.models import DatasetMetaData, FileMetaData
+
+TAGS_DATASETS: list[str | Enum] = ["datasets"]
+TAGS_FILES: list[str | Enum] = ["files"]
+TAGS_HEALTH: list[str | Enum] = ["health"]
+TAGS_LOCATIONS: list[str | Enum] = ["locations"]
+TAGS_TASKS: list[str | Enum] = ["tasks"]
+TAGS_SIMCORE_S3: list[str | Enum] = ["simcore-s3"]
+
+
+app = FastAPI(
+    redoc_url=None,
+    description="API definition for simcore-service-storage service",
+    version="0.3.0",
+    title="simcore-service-storage API",
+    contact={"name": "IT'IS Foundation", "email": "support@simcore.io"},
+    license_info={
+        "name": "MIT",
+        "__PLACEHOLDER___KEY_url": "https://github.com/ITISFoundation/osparc-simcore/blob/master/LICENSE",
+    },
+    servers=[{"description": "API server", "url": "/v0"}],
+    openapi_tags=[
+        {"name": x}
+        for x in list(
```

Comment:
this should work just by concatenating the lists, i.e. using `+` operator on TAGS_... lists!